### DISCUSSION TITLE
in the stock display, is it able to get the stock value instead of its unit purchase price?

### DISCUSSION THREAD

**Discussion by anonkwe** [inventree/InvenTree#4465] [Q&A] [answered]
Is it possible to multiply stock's unit purchase price by the real in stock quantity/count in order to get a real inventory value, otherwise, this unit purchase price is kind of useless. is it possible is there a modification in the code i can do to make it happen fast?

![image](https://user-images.githubusercontent.com/61838766/223670735-22697e65-5942-42fc-aaba-6b7d2524446c.png)

**Comment by matmair** [comment]
The purchase price is consistently the unit price, I am strongly against changing this and making it confusing.

Use a report to get the total purchase value of your items.

**Reply by anonkwe** [reply]
How? Is there any tool in place? Do you guys have any doc summing up every field we can use to pull out reports?

**Reply by SchrodingersGat** [reply]
The closest we have is https://docs.inventree.org/en/latest/report/report/

**Comment by SchrodingersGat** [comment, marked as answer]
It is imperative that we keep unit pricing which is independent of the quantity field. However it would be possible to add a new column to the table which calcules "value" based on `purchase_price x quantity`

**Reply by anonkwe** [reply]
This would be great and would make much more sense to have an overview of your in house stock and its value, is there anything i can tweak in the code meanwhile?

**Comment by winzkigermany** [comment]
Not too fast there, in `purchase_price x quantity` - which `purchase_price` ?
the last? the lowest? the highest? the weighted mean price?
I'm currently hunting a bug (or red herring) in the stocktake min/max value calculation, and in my opinion, calculating the value of stock is simple, yet not that simple. ;-)

**Reply by matmair** [reply]
The price at which the stock was purchased aka purchase price

**Reply by matmair** [reply]
The is a literal field on stock @winzkigermany

**Reply by SchrodingersGat** [reply]
@winzkigermany where purchase price is available for a specific stock, that is what is used here. This comes normally from the purchase order information. 

e.g. "this reel of capacitors was purchased at a unit price of $0.02" vs "that reel of capacitors was purchased at a unit price of $0.15"
